File: benches/mutations/variants/python/seed1.t3-loop-swap.py
def summarize_scores(entries, threshold):
    summaries = []
    running_total = 0
    for i in range(len(entries)):
        label, score = entries[i]
        if score < threshold:
            continue
        running_total += score
        if score >= 90:
            grade = "excellent"
        elif score >= 50:
            grade = "adequate"
        else:
            grade = "poor"
        summaries.append(label + ": " + str(score) + " (" + grade + ")")
    if running_total > 250:
        summaries.append("aggregate: high")
    return summaries

File: benches/mutations/variants/rust/seed1.t3-loop-swap.rs
pub fn summarize_scores(entries: &[(String, i64)], threshold: i64) -> Vec<String> {
    let mut summaries = Vec::new();
    let mut running_total = 0;
    let mut i = 0;
    while i < entries.len() {
        let (label, score) = &entries[i];
        i += 1;
        if *score < threshold {
            continue;
        }
        running_total += score;
        let grade = if *score >= 90 {
            "excellent"
        } else if *score >= 50 {
            "adequate"
        } else {
            "poor"
        };
        summaries.push(format!("{label}: {score} ({grade})"));
    }
    if running_total > 250 {
        summaries.push(String::from("aggregate: high"));
    }
    summaries
}
